src/rust/catpowder/win/runtime.rs

```diff
@@ -72,9 +74,16 @@ impl SharedCatpowderRuntime {
         for queueid in 0..queue_count {
             rx_rings.push(RxRing::new(&mut api, Self::RING_LENGTH, ifindex, queueid as u32)?);
         }
-        trace!("Created {} RX rings.", rx_rings.len());
+        trace!("Created {} RX rings on interface {}", rx_rings.len(), ifindex);
 
-        Ok(Self(SharedObject::new(CatpowderRuntimeInner { api, tx, rx_rings })))
+        let vf_queue_count = deduce_rss_settings(&mut api, vf_if_index)?;
+        let mut vf_rx_rings= Vec::with_capacity(vf_queue_count as usize);
+        for queueid in 0..vf_queue_count {
+            vf_rx_rings.push(RxRing::new(&mut api, Self::RING_LENGTH, vf_if_index, queueid as u32)?);
+        }
+        trace!("Created {} RX rings on interface {}.", vf_rx_rings.len(), vf_if_index);
+
+        Ok(Self(SharedObject::new(CatpowderRuntimeInner { api, tx, rx_rings, vf_rx_rings })))
```

Comment:
Could you please run the formatter on this code? If you are using vscode it can automatically do this for you.